C++/include/algorithm/dynamic_programming/weighted_activity_selection.hpp

```diff
@@ -0,0 +1,125 @@
+/*
+    Weighted activity selection problem
+    --------------------
+	Given N activities where every activity is represented by 
```

Comment:
Make same indentation as rest of block comment.

Reply:
And language might be more clear as: 

"Given N activities where each activity is represented by start time, finish time, and weight (or associated value), find the subset of activities with maximum weight such that no two activities overlap."